### DISCUSSION TITLE
FirebaseAnalytics should be open source

### DISCUSSION THREAD

**Discussion by sethfri** [firebase/firebase-ios-sdk#9989] [Pitches]
`FirebaseAnalytics` is currently only distributed as an XCFramework, which makes it difficult to see what Google is doing under the hood in our apps without plugging the SDK into [Hopper](https://www.hopperapp.com/) and attempting to reverse engineer it.

It would be great if Google would open source `FirebaseAnalytics` to make it easier for developers to inspect the code we are pulling into our apps.

Thank you!

**Comment by paulb777** [comment]
Thanks for the pitch. There are no current plans to open source FirebaseAnalytics. However, please keep voting this up to help us increase the priority.
